server/service.go

```diff
@@ -41,6 +42,16 @@ func respondError(w http.ResponseWriter, code int, message string) {
 	}
 }
 
+func respondOK(w http.ResponseWriter, result any) error {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	if err := json.NewEncoder(w).Encode(result); err != nil {
+		return err
+	}
+	return nil
```

Comment:
```suggestion
	return json.NewEncoder(w).Encode(result)
```

could simplify like this